server/views/pages/signedOut.njk

```diff
@@ -0,0 +1,18 @@
+{% extends "partials/layout.njk" %}
+
+{% set pageTitle = "You have signed out" %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-l">{{ pageTitle }}</h1>
+
+      <p>
+        You can book a visit or view your bookings by
+        <a href="https://signin.account.gov.uk/sign-in-or-create">signing into the service using GOV.UK One Login</a>.
```

Comment:
```suggestion
        <a href="/sign-in">signing into the service using GOV.UK One Login</a>.
```
You've used the link in the ticket - but that was wrong! (I've updated the ticket). To sign in to the service again, just `/sign-in` should do and that will redirect user to OL.